src/alpaka/plugin-PixelTriplets/alpaka/CAHitNtupletGeneratorKernels.cc

```diff
@@ -112,10 +110,6 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
                                                       alpaka::getPtrNative(device_nCells_)));
     }
 
-#ifdef GPU_DEBUG
-    alpaka::wait(queue);
-#endif
```

Comment:
I was thinking about it this morning, and I'd say "_maybe_":
  - if we want to make all CUDA-related operations synchronous _e.g._ for debugging, it would be simpler to change the `queue` used to submit the work to a synchronous one
  - if we want to keep a finer granularity, enabling the waits and checks for a single source file at a time, sure, we can keep them